code/modules/uplink/uplink_items/stealthy_tools.dm

```diff
@@ -18,6 +18,14 @@
 	item = /obj/item/card/id/advanced/chameleon
 	cost = 2
 
+/datum/uplink_item/stealthy_tools/agent_card_advanced
+	name = "Advanced Agent Identification Card"
+	desc = "Created by Cybersun Industries to be the ultimate for field operations, this upgraded Agent ID \
+	comes with all the fluff of the original, but with an upgraded microchip - allowing for the storage of all \
+	standard Nanotrasen access codes in one conveinent package. Now in glossy olive by default!"
```

Comment:
The other item descriptions in this file all have extra lines indented to the same level as the first one, so just for consistency:
```suggestion
	desc = "Created by Cybersun Industries to be the ultimate for field operations, this upgraded Agent ID \
			comes with all the fluff of the original, but with an upgraded microchip - allowing for the storage of all \
			standard Nanotrasen access codes in one conveinent package. Now in glossy olive by default!"
```